gems/pending/Gemfile

```diff
@@ -33,7 +33,8 @@ gem "net-sftp",                "~>2.1.2",           :require => false
 gem "nokogiri",                "~>1.6.8",           :require => false
 gem "openscap",                "~>0.4.3",           :require => false
 gem "openshift_client",        "=1.2.0",            :require => false
-gem "ovirt",                   "~>0.11.0",          :require => false
+gem "ovirt",                   "~>0.13.0",          :require => false
+gem "parallel",                "~>1.9",             :require => false # For OvirtInventory
```

Comment:
It's also bundled in via azure-armrest.  I think the comment should stay for now since that is the reason i put it in this Gemfile.  If the Ovirt provider is split out to its own gem, that dependency should go with it.